.isort

```diff
@@ -0,0 +1,10 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=true
+force_grid_wrap=0
+use_parentheses=true
+ensure_newline_before_comments=true
+line_length=88
+force_single_line=true
+order_by_type=true
+lines_between_types=1
```

Comment:
this file doesn't add any value to the code unless you're using a specific formatter